src/Http/WebUtilities/src/HttpResponseStreamWriter.cs

```diff
@@ -127,6 +127,30 @@ public override void Write(char[] values, int index, int count)
             }
         }
 
+        public override void Write(ReadOnlySpan<char> values)
+        {
+            if (_disposed)
+            {
+                throw new ObjectDisposedException(nameof(HttpResponseStreamWriter));
+            }
+
+            int written = 0;
+            while (written < values.Length)
```

Comment:
nit: save values.Length in a local rather than checking every iteration of the loop.